docs/BaremetalHost_ProvisioningState.dot

```diff
@@ -12,6 +12,7 @@ digraph BaremetalHost {
     ExternallyProvisioned [label="Externally\nProvisioned"]
 
     Registering -> Inspecting [label="!externallyProvisioned && NeedsHardwareInspection()"]
+    Registering -> Preparing [label="!externallyProvisioned && !NeedsHardwareInspection()"]
```

Comment:
`!NeedsHardwareInspection()` isn't actually what we're checking here though, it's `inspectionDisabled()`.